app/views/participants/validations/nino.html.erb

```diff
@@ -0,0 +1,17 @@
+<% title = "What’s your National Insurance number?" %>
+<% content_for :title, title %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= title %></h1>
+    <%= form_for validation_form, url: { action: :update }, method: :patch do |f| %>
+      <%= f.govuk_text_field :nino,
+        width: "three-quarters",
+        label: { text: "National Insurance Number", class: "govuk-visually-hidden" },
+        hint: { text: "It’s on your National Insurance card, benefit letter, payslip or P60. For example, ‘QQ 12 34 56 C’."}
```

Comment:
If I enter the example NINO, I get an error message telling me it's in the wrong format - but it plays back the text as we display it in the example. Just checking that's the correct response please? Thanks

Reply:
Yes, the example NINO is not a valid NINO number - which is in accordance with  govuk styleguide for nino: https://design-system.service.gov.uk/patterns/national-insurance-numbers/